### DISCUSSION TITLE
年轻人的第一架战斗机

### DISCUSSION THREAD

**Discussion by GrunderIndustries** [share121/inter-knot#2858] [其他]
![image](https://imgsa.baidu.com/forum/pic/item/9c56808ba61ea8d307584417940a304e241f58d3.jpg)
# 你还在为以太适应体质差而烦恼吗!
# 你还在为空洞内以骸的威胁而害怕吗!
# 你想在空洞久待 但是补给不允许吗！
## 创立者工业隆重推出
# ADA-01 ADLER
## 有了它 您就能在高空虐杀以骸
# 有了它 您才是空洞最大的威胁！
## 没有隐身需求的情况下还可以挂载四门30mm快速机炮或者四联50mm慢速机炮提供强大的近距离支援能力
# 同时其挂点还可以挂载一种MPBM的衍生型号"SDBM" 拥有一发摧毁小型空洞的能力
### 驾驶舱内置抗侵蚀维生装置和小型冰箱 还装有视频播放器
# 其抗侵蚀能力相较于旧文明战斗机得到极大改善 可以在空洞内巡航至少168小时不被侵蚀
# 还可以在飞行途中收集空洞数据来获得大量"萝卜"以自由进出空洞
## 相较于ADF-01更加便宜 仅售50000000丁尼

*SDBM尚未正式发售 请等待后续消息
*创立者工业不对任何因用户操作不当导致的法律风险和人身安全负责
*该机不附赠除机炮外的任何武器系统
*请在下单前确认您有足够大小的机库和跑道再下单
*数据收集功能在开启状态下可能会导致航程下降
~~TMD创立者工业简直不是人 一篇破文改七八遍了加班到半夜还不饶了我~~

**Comment by OttoApocalypse01** [comment]
武器系统要自己搞啊，新艾利都一般人买了好像也用不太上，机炮对恶名以骸的杀伤力实在有限<img src="https://img-static.mihoyo.com/communityweb/upload/7a20b07305ccd63f8a69c3decd5c2bbb.gif" alt="摸头" width="75" >

**Reply by GrunderIndustries** [reply]
经测试 四联50MM一轮齐射可以轻易地撕碎恶名以骸的装甲

**Comment by weimaojun** [comment]
🥱
